src/Runner.jl

```diff
@@ -3,6 +3,8 @@ abstract type Runner; end
 
 export default_host_platform
 
+const AnyRedirectable = Union{Base.AbstractCmd, Base.TTY, IOStream, IOBuffer}
```

Comment:
I moved this because it didn't really make sense to have it in  `UserNSRunner` when it is used for `DockerRunner` as well.

Reply:
Yes, makes sense :+1: